.gitignore

```diff
@@ -1,7 +1,6 @@
 # dist/
 # IDE config
 .vscode
-.idea
```

Comment:
I generally try to commit user-agnostic JetBrains configuration files to the repository with the expectation that users have [these `JetBrains.gitignore` entries](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore) to their local [global gitignore](https://github.com/github/gitignore/tree/master/Global#globally-useful-gitignores). This enables shared project-based IDE configurations which I find useful. This explains the committed `.idea/` files below.

If you're open to this, I can add something about this to the README.